libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator+Internal.h

```diff
@@ -39,7 +39,6 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, strong, readonly, nullable) NSURLSession *session;
 @property (nonatomic, strong , nullable) NSMutableData *responseData;
 @property (nonatomic, assign) BOOL initialRequestLoaded;
-@property (nonatomic, assign) BOOL domainUpdated;
```

Comment:
This one was to make sure it didn't loop but with the more specific URL and another tweak the case is already avoided